### DISCUSSION TITLE
Refund Request

### DISCUSSION THREAD

**Discussion by Unlimitedtang** [aclap-dev/video-downloadhelper#1232] [Help and Q&A]
### Your question

I downloaded the Google Chrome plug-in, but Google Chrome cannot be used. I want a refund.

Email:109503292@qq.com
License ID: 590cca525c

### Addon & CoApp version and details

I downloaded the Google Chrome plug-in, but Google Chrome cannot be used. I want a refund.

Email:109503292@qq.com
License ID: 590cca525c

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
